### DISCUSSION TITLE
Prophet support in Time Series Module

### DISCUSSION THREAD

**Discussion by ngupta23** [pycaret/pycaret#1773] [FAQs]
# Overview

`pycaret` time series module now supports Facebook's [prophet](https://facebook.github.io/prophet/docs/quick_start.html) model support. This is an optional dependency so it has to be installed separately from the `pycaret` time series installation. Let's see how

# Code Snippets

```python
#### Install library (one time) ----
!pip install prophet
!pip install --pre pycaret

#### Load data ----
from pycaret.datasets import get_data
y = get_data("airline")

#### Setup experiment ----
from pycaret.time_series import TSForecastingExperiment
exp = TSForecastingExperiment()
exp.setup(data=y, fh=12, session_id=42)

#### Initial Training ----
model = exp.create_model("prophet")
exp.plot_model(model)
```

![image](https://user-images.githubusercontent.com/33585645/139174902-5bf6e1f3-245f-4cc2-9a71-d5ab4ea8600e.png)

```python
#### Tune Model ----
tuned_model = exp.tune_model(model)
exp.plot_model(tuned_model)
```

![image](https://user-images.githubusercontent.com/33585645/139174915-e43069cb-be54-4e4b-b507-3bf052c0d1b9.png)

# Notebook

Google Colab notebook can be **[found here](https://gist.github.com/ngupta23/89a47f5ed75021b6f103c551eba9e5c9).**

**Comment by moezali1** [comment]
PyCaret 4.0 sunset the legacy time-series module (sktime/prophet integration), so this FAQ no longer maps to a supported code path.

If you still need time-series forecasting, use sktime / prophet / statsforecast directly — those projects have surpassed what the PyCaret wrapper offered.

For 4.0 docs, see https://pycaret.org/docs and the cleanup notice at https://github.com/pycaret/pycaret/issues/4177.

Closing as outdated.
